### PR TITLE
Fix Qt4 compilation

### PR DESCRIPTION
**Pull request by ksyy**
Linked to [Issue 115](https://github.com/Gepetto/gepetto-viewer-corba/issues/115)

### CODE REVIEW

File: include/gepetto/gui/fwd.hh [hunk 1]
```diff
@@ -19,8 +19,12 @@
 
 #include <vector>
 
-#include <QtGui/qopengl.h>
+#include <QtGlobal>
 #include <QtGui>
+#if QT_VERSION >= 0x050000
+    #include <QtGui/qopengl.h>
+#endif
```
Comment (jmirabel) [inline include/gepetto/gui/fwd.hh]: @jcarpent does the order matter here ?
I remember that it Qt5 and OpenGL includes are a bit touchy.

Comment (jcarpent) [reply include/gepetto/gui/fwd.hh]: `#include <QtGui>`should be included before `#include <QtGui/qopengl.h>`

Comment (jmirabel) [reply include/gepetto/gui/fwd.hh]: But you did the opposite here: https://github.com/Gepetto/gepetto-viewer-corba/commit/3dc7f167ec9c7e3bca6c54180bf153dbdeb5b613

Comment (jcarpent) [reply include/gepetto/gui/fwd.hh]: I meant the reverse, sory.

Comment (jmirabel) [reply include/gepetto/gui/fwd.hh]: Ok, thanks.

**Comment by jmirabel** [comment]
I merged that by hand and updated #114 . See 65b8fb8

**Comment by jmirabel** [comment]
Thanks

**Pull request closed without merging**